_rmd/2019-03-10-RPyGeo-1_0_0.Rmd

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: "RPyGeo 1.0.0"
+date:  "`r format(Sys.time(), '%d %B, %Y')`"
+comments: true
+author: Marc Becker
+categories: r
+---
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
+
+[DOWNLOADHERE]
+
+RPyGeo 1.0.0 has been released on CRAN. The RPyGeo package establishes an interface to the geoprocessing tools of ArcGIS from within R. ArGIS is a geographic information system developed by Esri. 
+
+The new RPyGeo package accesses the ArcPy Python side-package via the [reticulate](https://cran.r-project.org/web/packages/reticulate/index.html) package. ArcGIS provides almost all of it's geoprocessing tools via ArcPy. With RPyGeo these functions are available in R. Additonally RPyGeo provides utilitiy functions to create a seamless workflow between ArcGIS and R. 
+
+In 2015 Esri released the R-ArcGIS Bridge, a software that also connects ArcGIS and R. Essentially, the R-ArcGIS Bridge offers read, write and conversion functions to transfer data from ArcGIS to R and vice versa. The idea is to use the large number of R packages to solve spatial problems, which cannot be solved with ArcGIS alone. R scripts can be run as geospatial scripts with an interactive user inferface from within ArcGIS. However, no functionality is added for the R user. RPyGeo is able to run the geoprocessing tools of ArcGIS directly from the R session.      
```

Comment:
- link to R-ArcgGIS bridge

> "The idea is to use the large number of R packages to solve spatial problems" 

-> sth like "to use R to add additional capabilities for solving spatial problems"

> "R scripts can be run as geospatial scripts"

-> put "geospatial scripts" in parantheses or explain better what is meant by this in the context of ArcGIS

> "However, no functionality is added for the R user. RPyGeo is able to run the geoprocessing tools of ArcGIS directly from the R session."

 -> do you want to showcase a difference here? this does not become clear. Maybe add a bit more detail.